roaster-selenide/roaster-selenide-api/src/main/java/hu/icellmobilsoft/roaster/selenide/BaseSelenideTestCase.java

```diff
@@ -0,0 +1,64 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.roaster.selenide;
+
+import java.util.Collections;
+import java.util.Set;
+
+import javax.enterprise.inject.spi.Extension;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.openqa.selenium.WebDriver;
+
+import com.codeborne.selenide.WebDriverRunner;
+
+import hu.icellmobilsoft.roaster.weldunit.BaseWeldUnitType;
+
+/**
+ * Base class of Selenide TestCases, what is contained Selenide configuration
+ *
+ * @author speter555
+ * @since 0.2.0
+ */
+public class BaseSelenideTestCase extends BaseWeldUnitType {
+
+    @Override
+    protected Set<Class<? extends Extension>> listOfExtensionsToAddWeld() {
+        return Collections.singleton(SelenideConfigExtension.class);
+    }
+
+    /**
+     * Before each all test, setup webdriver if necessary
+     */
+    @BeforeEach
+    public void setupWebDriver() {
+        WebDriver webDriver = WebDriverRunner.getWebDriver();
```

Comment:
ha ez static beallitas, akkor valoszinu rahatassal van az osszes futasi tesztre. Ez viszont akkor egy konzponti beallitas kellene hogy legyen, nem egyenkent a tesztekre

Reply:
Kiveszem ezt, és magát a felbontás beállítását kicsit tovább gondolom, mert arra szükség van az egyik projekten globálisan, és ha kell még ilyen config akkor azokat majd a beállításoknál tovább bővíthetjük.